application-services/custom/aws-export/README.md

```diff
@@ -0,0 +1,29 @@
+# App-functions-aws
+
+App-functions-aws is a sample provided to let the user send mqtt messages to Amazon AWS using SSL
+
+# Configuration
+
+Client certificates (crt and key) can be copied in the folder certs
+
+The address of the AWS host and the path to the certificates can be found in the configuration.toml:
+
+[ApplicationSettings]
+
+AwsIoTMQTTHost      = "***.iot.us-west-2.amazonaws.com"
+
+AwsIoTMQTTPort      = "8883"
+
+
+MQTTCert            = "./certs/user.client.crt"
+
+MQTTKey             = "./certs/user.client.key"
+
+SkipCertVerify	    = "false"
+
+# Important note
+
+If SkipCertVerify is true, TLS accepts any certificate
+presented by the server and any host name in that certificate.
+In this mode, TLS is susceptible to man-in-the-middle attacks.
+This should be used only for testing. [golang reference](https://golang.org/pkg/crypto/tls/)
```

Comment:
format looks a bit weird. it changes to bold font amidst of line 13

Reply:
looks like `"***.iot.us-west-2.amazonaws.com"` is causing that.

Reply:
It renders fine in browser, this is just a GitHub issue. No change needed.

Reply:
ok, good to hear that :->